src/app/leaves/add/components/LeaveRequestFormBuilder.tsx

```diff
@@ -99,7 +112,7 @@ export const LeaveRequestFormBuilder = () => {
       </h1>
       <Form {...form}>
         <form
-          data-testid="form"
+          data-testid="validForm"
```

Comment:
This is a weird id. Should represent form name.

Reply:
Fixed